.gitlab/image.yml

```diff
@@ -1,82 +1,83 @@
 .use_gko-nocuda-gnu9-llvm8:
-  image: localhost:5000/gko-nocuda-gnu9-llvm8
+  image: ginkgohub/cpu:gnu9-llvm8
   tags:
     - private_ci
     - cpu
+    - amd
 
 .use_gko-nocuda-gnu9-llvm8-intel:
-  image: localhost:5000/gko-nocuda-gnu9-llvm8
+  image: ginkgohub/cpu:gnu9-llvm8-intel2020
   tags:
     - private_ci
     - cpu
-    - cuda
+    - controller
 
 .use_gko-cuda90-gnu5-llvm39:
-  image: localhost:5000/gko-cuda90-gnu5-llvm39
+  image: ginkgohub/cuda:90-gnu5-llvm39
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda91-gnu6-llvm40:
-  image: localhost:5000/gko-cuda91-gnu6-llvm40
+  image: ginkgohub/cuda:91-gnu6-llvm40
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda92-gnu7-llvm50-intel2017:
-  image: localhost:5000/gko-cuda92-gnu7-llvm50-intel2017
+  image: ginkgohub/cuda:92-gnu7-llvm50-intel2017
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda100-gnu7-llvm60-intel2018:
-  image: localhost:5000/gko-cuda100-gnu7-llvm60-intel2018
+  image: ginkgohub/cuda:100-gnu7-llvm60-intel2018
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda101-gnu8-llvm7-intel2019:
-  image: localhost:5000/gko-cuda101-gnu8-llvm7-intel2019
+  image: ginkgohub/cuda:101-gnu8-llvm7-intel2019
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda101-gnu8-llvm10-intel2019:
-  image: localhost:5000/gko-cuda101-gnu8-llvm10-intel2019
+  image: ginkgohub/cuda:101-gnu8-llvm10-intel2019
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
+    - cpu
 
 .use_gko-cuda102-gnu8-llvm8-intel2019:
-  image: localhost:5000/gko-cuda102-gnu8-llvm8-intel2019
+  image: ginkgohub/cuda:102-gnu8-llvm8-intel2019
   tags:
     - private_ci
-    - cuda
-    - gpu
+    - controller
```

Comment:
So, both the FH2 and the bwuni have the controller tags ?

Reply:
yes, since I'm using only the controllers now